mantidproject/mantid
# Vesuvio - VesuvioCorrections test to system test from unit test

The VesuvioCorrections unit test has been changed to be a unit test due to it's long execution time

**To test:**
- Ensure that the VesuvioCorrectionsTest is a system test and passes.
- Ensure that the VesuvioCorrectionsTest (that is a unit test) no longer exists.

Fixes #15786.

_Internal system test change: Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
